digital-deck-server/services/cardShuffleService.js

```diff
@@ -0,0 +1,23 @@
+module.exports = {
+    /**
+     * Shuffles the cards in the deck
+     * 
+     * @param cards The deck of cards to be drawn from
+     * @return {json} The json containing the ids cards currently in the player's hand and the current deck
+     */
+    ShuffleCards: function(cards) {
+      if (cards.length>1) {
```

Comment:
I'm not sure if it's a good idea to modify the parameter directly and return it. I'd recommend creating a new variable, modify that, and then return that as the result.

Reply:
I agree.